types/index.d.ts

```diff
@@ -362,6 +362,11 @@ declare namespace RamdaAdjunct {
          */
         isRegExp(val: any): boolean;
 
+        /**
+         * Checks if value is `Set`.
+         */
+        isSet(val: any): boolean;
```

Comment:
```ts
isSet(val: any): val is Set

Reply:
I think I have done so now. I had to do `val is Set<any>` due to it requiring a type

Reply:
Yeah, that's good enought.